region/maridia/inner-pink/Aqueduct.json

```diff
@@ -1281,7 +1281,7 @@
       "entranceCondition": {
         "comeInRunning": {
           "speedBooster": true,
-          "minTiles": 2.4375
+          "minTiles": 0.4375
         }
       },
       "requires": [
```

Comment:
not sure why the 25 tile shinecharge is on this strat; I guess it is to signify that if you dont know that you need to start close to the transition, then you will not hold dash immediately when starting to run.